src/Cake.Issues/Serialization/SerializableIssueV2Extensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace Cake.Issues.Serialization
+{
+    using System;
+
+    /// <summary>
+    /// Extensions for <see cref="SerializableIssueV2"/>.
+    /// </summary>
+    internal static class SerializableIssueV2Extensions
+    {
+        /// <summary>
+        /// Converts a <see cref="SerializableIssueV2"/> to an <see cref="Issue"/>.
+        /// </summary>
+        /// <param name="serializableIssue">Issue which should be converted.</param>
+        /// <returns>Converted issue.</returns>
+        internal static Issue ToIssue(this SerializableIssueV2 serializableIssue)
+        {
+#pragma warning disable SA1123 // Do not place regions within elements
+            #region DupFinder Exclusion
```

Comment:
Why a region in such a small class?

Reply:
@christianbumann This is the only way to exclude part of a file from dup finder, which I'm aware of (would be happily use another one if available)